apps/examples/log_dump/log_dump_main.c

```diff
@@ -42,13 +44,20 @@ int log_dump_main(int argc, char *argv[])
 {
 	int ret = 1;
 	char buf[READ_BUFFER_SIZE];
-	int fd = open("/proc/logsave", O_RDONLY);
+	int fd = open("/proc/logsave", O_RDWR);
 
 	if (fd < 0) {
 		printf("Please ensure that procfs is mounted\n");
 		return 0;
 	}
 
+	if (write(fd, LOGDUMP_SAVE_START, strlen(LOGDUMP_SAVE_START) + 1) < 0)
+	{
```

Comment:
This example is for printing the saved logs. So if we enable logs just before printing and disable it right after, I don't think we can see any logs in the output. Is this intended?

Reply:
Frankly, the log would be saved by default after boot up.
You can see the [os/kernel/log_dump/log_dump.c](https://github.com/Samsung/TizenRT/pull/5505/files#diff-421a0070940db6407c73bb1b24551470aeb61ba3ddceeaf10e1b52a5782a2274):105.
But to show the usage, I intentionally added start in here.
Anyway, we can see the logs as you expected.